storybook/src/components/document-component/document-component.core-stories.ts

```diff
@@ -18,7 +22,7 @@ const meta: Meta = {
 
 export default meta;
 
-const { Default, Demo, IMRO } = documentComponentStories(
+const { Default, Demo, DemoInhoudsopgave, IMRO } = documentComponentStories(
```

Comment:
De twee modussen zijn gelijk aan elkaar. Zullen we die gelijkwaardigheid in de demo's naar voren laten komen?

```suggestion
const { Default, DemoContents, DemoInhoudsopgave, IMRO } = documentComponentStories(
```

of

```suggestion
const { Default, Contents, Inhoudsopgave, IMRO } = documentComponentStories(
```

Reply:
Heb gekozen voor de 2e suggestie